### ISSUE TITLE
Broken links in Understanding/Techniques

### ISSUE THREAD

**Issue by momdo** [w3c/wcag#269]
According to following links:
- https://validator.w3.org/checklink?uri=https%3A%2F%2Fwww.w3.org%2FTR%2F2016%2FNOTE-UNDERSTANDING-WCAG20-20161007%2Fcomplete.html&hide_type=all&depth=&check=Check
- https://validator.w3.org/checklink?uri=https%3A%2F%2Fwww.w3.org%2FTR%2F2016%2FNOTE-WCAG20-TECHS-20161007%2Fgeneral.html&hide_type=all&depth=&check=Check
- https://validator.w3.org/checklink?uri=https%3A%2F%2Fwww.w3.org%2FTR%2F2016%2FNOTE-WCAG20-TECHS-20161007%2Fcss.html&hide_type=all&depth=&check=Check
- https://validator.w3.org/checklink?uri=https%3A%2F%2Fwww.w3.org%2FTR%2F2016%2FNOTE-WCAG20-TECHS-20161007%2Faria.html&hide_type=all&depth=&check=Check
- https://validator.w3.org/checklink?uri=https%3A%2F%2Fwww.w3.org%2FTR%2F2016%2FNOTE-WCAG20-TECHS-20161007%2Fflash.html&hide_type=all&depth=&check=Check
- https://validator.w3.org/checklink?uri=https%3A%2F%2Fwww.w3.org%2FTR%2F2016%2FNOTE-WCAG20-TECHS-20161007%2Fsilverlight.html&hide_type=all&depth=&check=Check
- https://validator.w3.org/checklink?uri=https%3A%2F%2Fwww.w3.org%2FTR%2F2016%2FNOTE-WCAG20-TECHS-20161007%2Fpdf.html&hide_type=all&depth=&check=Check
- https://validator.w3.org/checklink?uri=https%3A%2F%2Fwww.w3.org%2FTR%2F2016%2FNOTE-WCAG20-TECHS-20161007%2Ffailures.html&hide_type=all&depth=&check=Check

A lot of links are broken.

See also #268.

**Comment by fstrr** [comment]
The links referenced are all from 2016 and are:

1. [W3C Working Group note from 2016 on how to understand WCAG 2.0](https://www.w3.org/TR/2016/NOTE-UNDERSTANDING-WCAG20-20161007/complete.html).
2. [General Techniques For WCAG 2.0](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/general.html).
3. [CSS Techniques For WCAG 2.0](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/css.html).
4. [ARIA Techniques For WCAG 2.0](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/aria.html).
5. [Flash Techniques For WCAG 2.0](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/flash.html).
6. [Silverlight Techniques For WCAG 2.0](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/silverlight.html).
7. [PDF Techniques For WCAG 2.0](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/pdf.html).
8. [Failures For WCAG 2.0](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/failures.html).

I'm going to assume that these don't need updating, so am going to close this issue. If they do need updating, please re-open the issue (and help with the fixing?).
